test/parallel/test-eslint-inspector-check.js

```diff
@@ -12,12 +12,18 @@ const message = 'Please add a skipIfInspectorDisabled() call to allow this ' +
 new RuleTester().run('inspector-check', rule, {
   valid: [
     'foo;',
-    'common.skipIfInspectorDisabled(); require("inspector");'
+    'common.skipIfInspectorDisabled(); require("inspector");',
+    'require("common");\n' +
+    'common.skipIfInspectorDisabled();\n' +
+    'require("inspector");'
```

Comment:
Please either use a single code line (I would do that as in the example above the added one) or indent the code so the concatenated part is deeper indented than the start of the string.